### PR TITLE
[Foundation, TextInput] Added some private APIs

### PR DESCRIPTION
**Pull request by PoomSmart**

### CODE REVIEW

File: Foundation/NSString+Private.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#import <Foundation/Foundation.h>
+
+@interface NSString (Private)
+
++ (NSString *)stringWithUnichar:(unsigned int)aChar;
++ (NSString *)_stringWithUnichar:(unsigned int)aChar;
```
Comment (kirb) [inline Foundation/NSString+Private.h:6]: Interesting, any idea what the difference is between underscore and no underscore? Would one be more preferable over the other (e.g. one just calls the other method) or do they have different implementations?

Comment (PoomSmart) [reply Foundation/NSString+Private.h:6]: Yeah, one of them calls the other. I will remove the underscored version.

**Comment by leptos-null** [comment]
According to `lldb`, `stringWithUnichar:` is also in `TIExtras`

```
(lldb) dis ...
TextInput`+[NSString(TIExtras) stringWithUnichar:]:
```

Unrelated: I originally considered the parameter to this method was a `unichar` due to the method name, however the parameter type is (correctly listed as) `unsigned int` while `typedef unsigned short unichar`. Adding this note here for reference

**Comment by PoomSmart** [comment]
I would say that it might be `UniChar` or `UTF32Char` or whatever that doesn’t match the method name. I personally use this method in my emoji tweak so I know exactly if the data type is correct or not. 😂

**Pull request merged**